### PR TITLE
More admin page schtuff

### PR DESCRIPTION
**Pull request by adam-hart**
Needs some dear loving but here's a start.

@craig-day @thebritican

### CODE REVIEW

File: components/Admin/Admin.js [hunk 1]
```diff
@@ -57,12 +62,22 @@ class Admin extends React.Component {
     })
   }
 
+  renderEditPage = (location) => {
+    this.setState(Object.assign({}, this.state, {showEditPage: true, currentLocation: location}))
```
Comment (craig-day) [inline components/Admin/Admin.js]: I was wrong about this.  `setState` does a shallow merge so you don't need this crappy `Object.assign` foo. 

```js
  this.setState({ showEdidtPage: true, currentLocation: location })
```

File: components/Admin/Admin.js [hunk 2]
```diff
@@ -57,12 +62,22 @@ class Admin extends React.Component {
...
+  }
+
+  static propTypes = {
+    to: PropTypes.oneOfType([PropTypes.string, PropTypes.object])
+  };
```
Comment (craig-day) [inline components/Admin/Admin.js]: needed? Don't see it being used.

File: components/Admin/Admin.js [hunk 3]
```diff
@@ -87,7 +104,13 @@ class Admin extends React.Component {
           onNewFacility={this.handleNewFacility}
           onCategoryFilter={this.handleCategoryFilter}
         />
-        <LocationList locations={filteredLocations || locations} />
+        {showEditPage ? 
+          <LocationEdit location={currentLocation}/> :
+          <div>
+            <CategoryFilter />
```
Comment (craig-day) [inline components/Admin/Admin.js]: This is not here right? This should just be part of `AdminTopBar`

Comment (adam-hart) [inline components/Admin/Admin.js]: Yup

**Comment by craig-day** [comment]
couple comments but suhweeeeeettt :+1:

**Pull request merged**